api/test-helpers.js

```diff
@@ -47,13 +47,25 @@ function makeConnectionWithResponse(t, options, loginResponse, callback) {
       callback(conn, ws);
     });
     // Reply to the login request.
-    const mergedLoginResponse = Object.assign(defaultLoginResponse, loginResponse);
-    ws.reply({response: mergedLoginResponse});
+    const generatedResponse = makeLoginResponse(loginResponse);
+    ws.reply(generatedResponse);
   });
   // Open the WebSocket connection.
   ws.open();
 }
 
+/**
+  Generate a login response mixing in the supplied overrides to the
+  defaultLoginResponse.
```

Comment:
Since defaultLoginResponse is an internal detail there is no point in putting it on the docstring. `mixing in the supplied overrides to a default standard login response`?

Reply:
done